bread/bread.py

```diff
@@ -105,17 +102,19 @@ def dispatch(self, request, *args, **kwargs):
         return super(BreadViewMixin, self).dispatch(request, *args, **kwargs)
 
     def get_template_names(self):
-        # Is there a template_name_pattern?
+        # Return Django Vanilla templates (app-specific), then
+        #        Customized templates (via Bread object or Django settings), then
+        #        Django Bread templates
+        vanilla_templates = super(BreadViewMixin, self).get_template_names()
+        default_templates = ['bread/%s.html' % self.template_name_suffix]
         if self.bread.template_name_pattern:
-            return [self.bread.template_name_pattern.format(
+            custom_templates = [self.bread.template_name_pattern.format(
```

Comment:
Can we change `default_templates` and `custom_templates` to just `default_template` and `custom_template` and not have them be lists? Logically there's only ever going to be one value for each of these.